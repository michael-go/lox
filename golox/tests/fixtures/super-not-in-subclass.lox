class Eclair {
  cook() {
    super.cook();
    print "Pipe full of crème pâtissière.";
  }
}

super.notEvenInAClass();
