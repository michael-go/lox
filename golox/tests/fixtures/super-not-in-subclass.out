# exit code: 1
# stdout:
[line 3] Error: : Can't use 'super' in a class with no superclass.
[line 8] Error: : Can't use 'super' outside of a class.
